### DISCUSSION TITLE
does fortls work with `use mpi`

### DISCUSSION THREAD

**Discussion by TomMelt** [fortran-lang/fortls#345] [Q&A]
When I include mpi using the syntax below, fortls cannot find the module. 

```fortran
use mpi
```

I get `Module "mpi" not found in project` even though I have added the location of mpi.mod and the mpi header files using:

```
{
  "include_dirs": [
"/software/spack/opt/spack/linux-ubuntu22.04-skylake/gcc-11.4.0/openmpi-4.1.6-huwwnzgqj6cpssh37hgrd43o6y75sstf/include",
"/software/spack/opt/spack/linux-ubuntu22.04-skylake/gcc-11.4.0/openmpi-4.1.6-huwwnzgqj6cpssh37hgrd43o6y75sstf/lib",
],
}
```
I have also tried `source_dirs` but with no luck.

Am I doing something wrong?

FYI I am using version `fortls     2.13.0` installed via `pip`.

**Comment by gnikit** [comment]
fortls cannot read .mod files (since they are compiler and ABI dependant), it can read only Fortran sources. If you are using  VS Code then you should be specifying the the include path to the linter (i.e. the static analysis tool) which IS a compiler.

**Comment by TomMelt** [comment]
Ah I see. No I am using `vim`. Do you know of a simple way to get the same behaviour in `vim`?

**Reply by hansolsuh** [reply]
Sorry for pinging old thread, but have you figure this out?
